spec/index.html

```diff
@@ -596,7 +596,9 @@ <h2>Basic Concepts</h2>
     <section id="json-vs-yaml">
       <h3>JSON vs YAML comparison</h3>
 
-      <p>YAML is more flexible than JSON, as illustrated by comparison table below.</p>
+      <p>YAML is a superset of JSON, i.e. every valid JSON document is also a valid YAML document.
+        But YAML offers a number of extra features, chief amongst which is human readability due to the ability to use minimal punctuation.
+        YAML is more flexible than JSON, as illustrated by the comparison table below.</p>
```

Comment:
```suggestion
        But YAML offers a number of extra features, chief amongst which is human readability due to the ability to use minimal punctuation.</p>
        <p>YAML is more flexible than JSON, as illustrated by the comparison table below.</p>
```

Perhaps these could be on separate lines.